lumicks/pylake/channel.py

```diff
@@ -247,6 +262,91 @@ def downsampled_by(self, factor, reduce):
         raise NotImplementedError("Downsampling is not available for time tag data")
 
 
+class ContinuousCalibrated(Continuous, ForceCalibration):
```

Comment:
I would not reach for inheritance here. This implementation seems to double the number of classes and also duplicates a lot of code. It should be possible to just make `ForceCalibration` an attribute of the existing `Continuous` class.

Reply:
Yeah. I was actually doubting here.

Considering that we'd just have ForceCalibration as an internal, I also renamed its class's data as _src, since _calibration._calibration looks pretty horrible and unclear.

Do you think it makes more sense to have the assignment of the calibration attribute in the constructor, or via a separate attachment function?

The new implementation makes the named tuple feel a little redundant, since I construct ForceCalibration anyway.